### ISSUE TITLE
Describe when cross-references to graphics, figures, and tables are required

### ISSUE THREAD

**Issue by Miciah** [StyleGuides/WritingStyleGuide#120]
The guide cautions against excessive and non-descriptive cross-references and over-reliance on cross-references, but it does not say when cross-references are expected.

I am wondering (a) whether it is all right to have dangling (unreferenced) graphics, figures, or tables; (b) whether every figure etc. should be implicitly or explicitly mentioned at least once in the text; or (c) whether every figure etc. should be referenced explicitly by its label at least once.

Generally, I would expect (b) or (c), which is what I was taught in college, and some manuals say the same:

* "With a few exceptions—such as cover illustrations used to grab attention—graphics should be accompanied by clear references within your text" (_Technical Communication: A Practical Approach_ Pfeiffer, W.S., 2006).
* The APA Publication Manual states, "An informative table supplements—rather than duplicates—the text. In the text, refer to every table and tell the reader what to look for", and the manual further has a "Figure Checklist" with an item, "Are all figures mentioned in the text?".
* [Purdue OWL's MLA guide](https://owl.english.purdue.edu/owl/resource/747/14/) states, "The illustration label and number should always appear in two places: the document main text (e.g. see fig. 1) and near the illustration itself (Fig. 1)."

I found seemingly contradictory guidelines in the IBM Style Guide, which at one point seems to be saying (b): "Always introduce a figure with a sentence immediately preceding it, and use a period or a colon to end the sentence.  If you present the same type of information in two or more consecutive figures, you do not have to introduce each specific figure, but you must introduce the group of figures", but later states, "It is not necessary to use a cross reference to a figure or table in text if the figure or table is not referred to, or if it is referred to in such a way that the figure or table can be identified without a number", which looks like (a).

Could the Style Guide clarify this point?

**Comment by daobrien** [comment]
I think the first thing to clarify is the difference between a reference and a cross-reference.

You can refer to a figure or table that is in close proximity to the text, e.g., on the same page, and that is a reference. If you refer to the same figure or table from some other section of the book, either with a link or by referring to a page number and figure label, that is a cross-reference.

I think your comment (ending with "which looks like (a)") might confuse reference and cross-reference. The surrounding text might *refer* to the figure or table, but not by number. For example, the surrounding text might include "These protocols are listed in the following table." This *refers* to the table but does not include a cross-reference.

I agree that in most cases option (b) is preferred. Whether you use "Fig. 1" or some other format depends on several factors; the Technical Writing Style Guide is written in DocBook and uses XSL to define these formats. Other publication types have their own requirements.

I would avoid option (a).
I believe option (c) is only necessary for cross-references.

I'll review this section in the guide and determine what improvements might be possible. Thanks for the feedback.

**Comment by Miciah** [comment]
You're right, I was conflating references and cross-references.  With non-print media, the distinction becomes fuzzier, and "reference" has many meanings, so I thought I could use "cross-reference" as a more specific term, but it seems it is a less accurate term.  I'll amend the title of this issue.

However, with that clarification, this statement from the IBM Style Guide is all the more confusing: "It is not necessary to use a cross reference to a figure or table in text if the figure or table is not referred to, or if it is referred to in such a way that the figure or table can be identified without a number." Maybe "use a cross reference" in this context means having an anchor or label.  Otherwise, the statement (at least the first part, up to the "or") is self-evident.

In any case, your explanation is very helpful.  I had been looking for a guideline I could cite along the lines of, "Every figure or table should be referenced in the text", and I thought the guide was making a contrary statement, but I understand it was talking about something else now.

At this point, I would humbly suggest only (a) a positive statement about always referencing figures or tables and (b) brief definitions of, or an  explanation of the distinction between, a reference and a cross-reference.  Whether or not you decide these items belong in the guide, thank you for clearing up my misunderstanding!

**Comment by daobrien** [comment]
Thanks for your comments. I'll review this carefully and update the guide as appropriate.
